src/org/wojcieszko/chess/Tower.java

```diff
@@ -0,0 +1,23 @@
+package org.wojcieszko.chess;
+
+public class Tower {
```

Comment:
To już nie algorytmika, a inżynieria:
Tower może dziedziczyć po jakiejś ogólniejszej klasie `Piece` (BTW: właśnie się dopatrzyłem, że to jest Rook)
I wtedy inne figury mogłyby się różnić tylko tabelką ruchów i parametrem poSąsiedzku/DoKoncaPlanszy. Mniej więcej.